xml/obs_scm_bridge.xml

```diff
@@ -12,31 +12,34 @@
          xmlns:ns5="http://www.w3.org/2000/svg"
          xmlns:ns4="http://www.w3.org/1999/xhtml"
          xmlns:ns="http://docbook.org/ns/docbook">
-  <title>SCM Bridge [EXPERIMENTAL]</title>
+  <title>SCM Bridge</title>
 
   <sect1 xml:id="sec.obs.obs_scm_bridge">
     <title>SCM Bridge</title>
 
     <sect2 xml:id="sec.obs.obs_scm_bridge.setup.introduction">
       <title>Introduction</title>
 
-      <para>The SCM bridge allows to setup single packages or entire projects in any trusted SCM repositories. 
+      <para>The SCM bridge allows to setup single package or entire project sources to be stored and
+            maintained in any trusted SCM repositories. 
```

Comment:
It's a good idea, when introducing a new acronym into a text, to provide an expansion of the acronym. 

I'm guessing this is the first instance of "SCM" in the document's text, so we could change "SCM bridge" to "SCM (source control management) bridge" here?